master/static/srv/move_model.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE models SET workspace_id = $2
+WHERE name = $1
```

Comment:
i was thinking which one is better to use between model name or model id, so i looked into some code. 
other APIs take model name instead of id, and [this sql](https://github.com/determined-ai/determined/blob/master/master/static/srv/get_model.sql) also uses model name instead of id. 
just clarification: do we still wanna use id here?

Reply:
yeah sorry this is confusing, I am still in favor of ID especially when we are querying the DB. It would be OK to update get_model.sql as well

Reply:
`ModelFromIdentifier` func which uses `get_model.sql` inside is used in many places. I just update the `move_model.sql` for now with `currModel.Id`.  also move API still take `model_name` instead of id to follow the other APIs. we can change it in different pr cuz that would be a  big change

Reply:
:+1: